docs/examples/quickstart/quickstart-contract.yaml

```diff
@@ -11,22 +11,3 @@ materials:
   - name: vulnerabilities-report
     type: SARIF
     optional: true
-policies:
```

Comment:
but not this one, note that it's being used from the quickstart in the platform docs.

Reply:
Then it won't work for regular OSS users. Should we then provide two different contracts and distinguish them in the docs?

Reply:
I've created another contract without policies for opensource users.